baybe/surrogates/ngboost.py

```diff
@@ -1,11 +1,4 @@
-"""NGBoost surrogates.
-
-Currently, the documentation for this surrogate is not available. This is due to a bug
-in our documentation tool, see https://github.com/sphinx-doc/sphinx/issues/11750.
```

Comment:
Did you check that we now see the models in the documentation?

Reply:
Yes, all looking good:
<img width="1359" alt="image" src="https://github.com/user-attachments/assets/4d2f3989-f15d-4276-9148-6a10a5bb0816">